src/fidesctl/cli/commands/core.py

```diff
@@ -29,6 +29,33 @@
 @manifests_dir_argument
 @with_analytics
 def apply(ctx: click.Context, dry: bool, diff: bool, manifests_dir: str) -> None:
+    """
+    Validate local manifest files and persist any changes via the API server.
+    Deprecated in favor of `fidesctl push` command.
+    """
+
+    config = ctx.obj["CONFIG"]
+    taxonomy = _parse.parse(manifests_dir)
+    _apply.apply(
+        url=config.cli.server_url,
+        taxonomy=taxonomy,
+        headers=config.user.request_headers,
+        dry=dry,
+        diff=diff,
+    )
+
+
+@click.command()
+@click.pass_context
+@dry_flag
+@click.option(
+    "--diff",
+    is_flag=True,
+    help="Include any changes between server and local resources in the command output",
+)
+@manifests_dir_argument
+@with_analytics
+def push(ctx: click.Context, dry: bool, diff: bool, manifests_dir: str) -> None:
```

Comment:
is it possible to duplicate the command without a copy/pasta? I haven't tried it but I was hoping there'd be a clever way to do that, if not its fine, this command doesn't change much!

Reply:
after looking into some of the solutions you mentioned, it's not worth the eng time to figure this out before the deprecation, I'm good with how it is 👍